.github/workflows/sycl-nightly.yml

```diff
@@ -74,13 +74,6 @@ jobs:
             target_devices: opencl:cpu
             tests_selector: e2e
 
-          - name: Self-hosted CUDA
```

Comment:
Instead of removing this code, can we just comment it out?

Reply:
I think it's better to remove. If the runner is unrecoverable or nobody is willing to fix it then there is no reason to have the "dead" comments inside the repo.